kong/plugins/prometheus/exporter.lua

```diff
@@ -145,6 +147,21 @@ local function init()
                                           {"service", "route", "direction", "workspace"})
   end
 
+  -- AI mode
+  if ai_request then
+    metrics.ai_llm_requests = prometheus:counter("ai_llm_requests_total",
```

Comment:
just for my learning: what does db_name mean in this context? I saw it come from the cache field later.

Reply:
@fffonion  `db_name` is the name of the vector Database, I am going to also add the embedding drivers